src/go/gcsrunner/main/runner.go

```diff
@@ -68,7 +67,17 @@ func main() {
 	if err != nil {
 		glog.Fatal("PORT variable must be a valid port number.")
 	}
-
+	loopbackPort := os.Getenv("LOOPBACK_PORT")
+	if loopbackPort == "" {
+		loopbackPort = "8090"
+	}
+	lpNum, err := strconv.ParseUint(loopbackPort, 10, 32)
+	if err != nil {
+		glog.Fatal("LOOPBACK_PORT variable must be a valid port number.")
+	}
```

Comment:
It seem "PORT" and "LOOPBACK_PORT" are using the same code.  Can we use a function to share them?  I think it will make the code more readable.